tests/test_ext_marshmallow_field.py

```diff
@@ -318,6 +318,73 @@ def test_nested_field_with_property(spec_fixture):
     }
 
 
+def test_field2property_plock_spec_metadatas(spec_fixture):
+    breed = fields.Pluck(
+        CategorySchema,
+        "breed",
+        metadata={
+            "description": "Category breed",
+            "invalid_property": "not in the result",
+            "x_extension": "A great extension",
+        },
+    )
+    result = spec_fixture.openapi.field2property(breed)
+
+    spec_fixture.spec.components.schema("Category", schema=CategorySchema)
+    unplucked = spec_fixture.spec.components.schemas["Category"]["properties"]["breed"]
```

Comment:
I think I'd use `tests.utils.get_schema` to avoid relying on the internals of `Components`.

I admit we didn't do it 100% consistently in the tests.

I'm open to discussion and in any case this is not a blocker.

Reply:
I'll just use `tests.utils.get_schema()` so as not to increase the tech debt.